modules/plotting/properties/axisproperty.h

```diff
@@ -63,13 +63,15 @@ class IVW_MODULE_PLOTTING_API AxisProperty : public CompositeProperty {
     virtual AxisProperty* clone() const override;
     virtual ~AxisProperty() = default;
 
-    void setTitle(const std::string& title) {
-        caption_.title_.set(title);
-    }
-    void setLabelFormat(const std::string& formatStr) {
-        labels_.title_.set(formatStr);
-    }
-    void setRange(const dvec2& range) { range_.set(range); }
+    void setTitle(const std::string& title);
+    const std::string& getTitle() const;
+    void setLabelFormat(const std::string& formatStr);
```

Comment:
Add info about format, css style?

Reply:
not going to fix this here and now. Should be part of future PR, see #117